File: atest.rs
use std::io;

fn main() {
    let mut flag = 1;
    while flag == 1 {
        let mut write = String::new();
        println!("Write something:");
        let _ = io::stdin().read_line(&mut write);
        println!("You typed:{}", write);
        if write.as_mut_str() == "q\n" {
            flag = 0;
        }
    }
}

File: atest.scala
object multilang {




def main(args: Array[String]): Unit = {
  var flag = 1
  var write = ""
  while (  flag ==  1) {
    println(     "Write Something:")
    write = scala.io.StdIn.readLine("")
    println(     "You typed:" + write)
    if (      write ==     "q") {
      flag=0
    }
  }
}
}

